asyncapi/json-schemas/pe/sales/sla/sales-sla.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.ruter.no/adt/ota/api/v3.0/pe/sales/diagnostics/sales-diagnostics.json",
+  "type": "object",
+  "title": "SalesSla",
+  "required": [
+    "eventTimestamp",
+    "traceId",
+    "spanId",
```

Comment:
denne er optional i `current_stop`, burde den ikke være optional her og da? Hva om vi ikke får en `spanId` fra `current_stop`?

Reply:
Godt poeng. Den burde egentlig være required i current_stop også. 
Jeg er usikker på om vi skal gjøre begge required nå, eller begge optional nå og required for ADT4...
Jeg ønsker at det skal være enkelt å koble disse meldingene sammen, så kanskje er det ikke noen grunn til å vente. Særlig siden SLA-meldingen er helt ny, og current_stop er INTERNAL og dermed fristilt fra ADT-versjoner. Jeg tenker på det og tar en avgjørelse før merge.